sympy/integrals/tests/test_failing_integrals.py

```diff
@@ -246,3 +260,12 @@ def test_issue_7147():
 @XFAIL
 def test_issue_7109():
     assert not integrate(sqrt(a**2/(a**2 - x**2)), x).has(Integral)
+
+
+@XFAIL
+def test_integrate_Piecewise_rational_over_reals():
+    f = Piecewise(
+        (0,                                              t - 478.515625*pi <  0),
+        (13.2075145209219*pi/(0.000871222*t + 0.995)**2, t - 478.515625*pi >= 0))
+
+    assert integrate(f, (t, 0, oo)) == 15235.9375*pi
```

Comment:
If this test passed presumably it would be a close to rather than exactly equal situatiion. Maybe it should be something like `assert abs(result - expected) < 1e-16*pi` or something

Reply:
Yes, you are right. At the moment it is far from passing though. Actually a quite interesting test as it currently gives 0.0 as a result. However, integrating `13.2075145209219*pi/(0.000871222*t + 0.995)**2` from `478.515625*pi` to `oo` gives a non-zero answer (not the one stated though).

I can at least add a few comments there (or use the NS function...).